### ISSUE TITLE
No code-completion

### ISSUE THREAD

**Issue by tegrity** [dragos/dragos-vscode-scala#33]
It seems to me like I'm failing at installing the plugin properly. I don't see any error messages, but features like code-completion and errors as I type don't work for me. The only thing I see is syntax highlighting and the options to add sbt-resolvers when I hit ctrl+space.

What I did: 
I installed the plugin via the marketplace, told sbt to add ensime in the plugins.sbt and ran `sbt ensimeConfig`. I'm using Windows 10 and JAVA_HOME and SBT_HOME are set.  

Did I overlook something?

**Comment by dragos** [comment]
I don't think you missed anything. Do you see an `ensime-langserver.log` file in the project directory? Any hints in there?

**Comment by tegrity** [comment]
No hints in there:

> [2017-04-18 12:31:47,961] INFO org.github.dragos.vscode.Main$ - Starting server in c:\Users\simon\Documents\ScalaStuff
> [2017-04-18 12:31:47,961] INFO org.github.dragos.vscode.Main$ - Starting server in c:\Users\simon\Documents\ScalaStuff
> [2017-04-18 12:31:47,971] INFO org.github.dragos.vscode.Main$ - Classpath: C:\Users\simon\.vscode\extensions\dragos.scala-lsp-0.1.3\coursier
> [2017-04-18 12:31:47,971] INFO org.github.dragos.vscode.Main$ - Classpath: C:\Users\simon\.vscode\extensions\dragos.scala-lsp-0.1.3\coursier
> [2017-04-18 12:31:48,276] INFO akka.event.slf4j.Slf4jLogger - Slf4jLogger started
> [2017-04-18 12:31:48,507] INFO org.github.dragos.vscode.EnsimeLanguageServer - Initialized with 7924, c:\Users\simon\Documents\ScalaStuff, ClientCapabilities()
> [2017-04-18 12:31:48,507] INFO org.github.dragos.vscode.EnsimeLanguageServer - Initialized with 7924, c:\Users\simon\Documents\ScalaStuff, ClientCapabilities()
> 

But there is a `pc.stdout.log` containing the following

> Exception in thread "pool-5-thread-1" java.lang.NoClassDefFoundError: com/google/common/base/Charsets
> 	at org.github.dragos.vscode.EnsimeLanguageServer$$anonfun$2.apply(EnsimeLanguageServer.scala:88)
> 	at org.github.dragos.vscode.EnsimeLanguageServer$$anonfun$2.apply(EnsimeLanguageServer.scala:88)
> 	at scala.util.Try$.apply(Try.scala:192)
> 	at org.github.dragos.vscode.EnsimeLanguageServer.org$github$dragos$vscode$EnsimeLanguageServer$$initializeEnsime(EnsimeLanguageServer.scala:87)
> 	at org.github.dragos.vscode.EnsimeLanguageServer.initialize(EnsimeLanguageServer.scala:74)
> 	at langserver.core.LanguageServer$$anonfun$1.apply(LanguageServer.scala:19)
> 	at langserver.core.LanguageServer$$anonfun$1.apply(LanguageServer.scala:16)
> 	at langserver.core.Connection$$anonfun$handleCommand$1.apply(Connection.scala:161)
> 	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24)
> 	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24)
> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
> 	at java.lang.Thread.run(Unknown Source)
> Caused by: java.lang.ClassNotFoundException: com.google.common.base.Charsets
> 	at java.net.URLClassLoader.findClass(Unknown Source)
> 	at java.lang.ClassLoader.loadClass(Unknown Source)
> 	at java.lang.ClassLoader.loadClass(Unknown Source)
> 	... 13 more

**Comment by tegrity** [comment]
Now it works for me. I've installed the update you made today and suddenly vsc told me, that there was no .ensime file (even though I'm pretty sure there was one). Well, i ran the command again and now it works. I don't really know what was going on, since I had the same problem on a second PC. Maybe the .ensime was in the wrong folder.
However, thanks for your help!

**Comment by dragos** [comment]
Thanks for digging it up. I just noticed yesterday the same compilation error (well, "class not found"). It turns out that Ensime removed a dependency to Guava, which we inherited and used. I updated the code on our side as well, and that's why yesterday's update started working again.

I should probably switch to a stable Ensime version (this is 2.0-SNAPSHOT), but as far as I know there hasn't been a 2.0 release yet (/cc @fommil).
